src/medCore/process/medRunnableProcess.cpp

```diff
@@ -52,11 +52,17 @@ dtkAbstractProcess * medRunnableProcess::getProcess()
 
 void medRunnableProcess::internalRun()
 {
-    if (d->process) {
-        if (d->process->update() == 0)
+    if (d->process)
+    {
+        int res = d->process->update();
+
+        if (res == 0)
```

Comment:
Just stating my opinion but for readability I don't think it's useful to have moved ```d->process->update() ```out of the ```if ```statement.